azurerm/internal/services/dns/tests/dns_zone_resource_test.go

```diff
@@ -80,6 +80,25 @@ func TestAccAzureRMDnsZone_withTags(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMDnsZone_withSOARecord(t *testing.T) {
```

Comment:
Could we test more? we should have a basic and complete config and tests. We should have a test going basic -> complete -> basic

Reply:
updated